cluster-provision/manifests/cni/calico/script.d/002-allow-ip-forwarding.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/env bash
+
+set -e
+
+patch -p1 --ignore-whitespace << 'EOF'
```

Comment:
Can you explain how this done please?

Reply:
It's suppose to patch manifest.yaml but the code here was not being executed, now it should be fine, let's re-review.